app/scripts/controllers/serviceInstance.js

```diff
@@ -204,6 +206,12 @@ angular.module('openshiftConsole')
         $scope.project = project;
         $scope.projectContext = context;
 
+        if ($scope.AEROGEAR_MOBILE_ENABLED) {
```

Comment:
`AEROGEAR_MOBILE_ENABLED` is originally set on `$rootScope`? Is it available on `$scope` too? Im unsure of the hierarchy

and should our check be `$scope.AEROGEAR_MOBILE_ENABLED && $scope.isMobileService`?